src/main/resources/snowflake-schema-crawler-columns.sql

```diff
@@ -0,0 +1,4 @@
+select TABLE_NAME, COLUMN_NAME, DATA_TYPE, IS_NULLABLE, COMMENT, ORDINAL_POSITION,
+       DECODE(DATA_TYPE, 'NUMBER', NUMERIC_PRECISION, 'TEXT', CHARACTER_MAXIMUM_LENGTH, 'TIMESTAMP_LTZ', DATETIME_PRECISION , 'TIMESTAMP_NTZ' ,DATETIME_PRECISION, 0) as PRECISION,
+       DECODE(DATA_TYPE, 'NUMBER', NUMERIC_SCALE,  0) as SCALE
+ from information_schema.columns where table_schema = '{{SCHEMA_NAME}}' AND table_name IN ({{TABLE_LIST}}) ;
```

Comment:
Please review this query that retrieves columns details

Reply:
Nice. I was going to correct your use of PRECISION above but it's right.